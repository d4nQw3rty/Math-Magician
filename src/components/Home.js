import React from 'react';

const Home = () => (
    <div>
      <h2>Welcome to our page!</h2>
      <p className={styles.p1}>
        Lorem ipsum, dolor sit amet consectetur adipisicing elit.
        Animi cumque sit quaerat cum,ratione soluta! Eveniet, magnam
        eum ea excepturi et temporibus laborum at officia!
      </p>
      <hr />
      <p className={styles.p2}>
        Lorem ipsum, dolor sit amet consectetur adipisicing elite.
        Animi cumque sit quaerat cum, ratione soluta! Eveniet, magnam
        eum ea excepturi et temporibus laborum at officia!
      </p>
    </div>
  );
  
  export default Home;