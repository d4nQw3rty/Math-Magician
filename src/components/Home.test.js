import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home.js';

describe ('Checking home deployment', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<Home/>).toJSON();
    expect(tree).toMatchSnapshot();
  })
})
