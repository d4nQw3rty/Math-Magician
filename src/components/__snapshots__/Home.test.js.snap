// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checking home deployment renders correctly 1`] = `
<div>
  <h2>
    Welcome to our page!
  </h2>
  <div
    className="home"
  >
    <p
      className="p1"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia, molestiae quas vel sint commodi repudiandae consequuntur voluptatum laborum numquam blanditiis harum quisquam eius sed odit fugiat iusto fuga praesentium optio, eaque rerum! Provident similique accusantium nemo autem. Veritatis obcaecati tenetur iure eius earum ut molestias architecto voluptate aliquam nihil, eveniet aliquid culpa officia aut! Impedit sit sunt quaerat, odit, tenetur error, harum nesciunt ipsum debitis quas aliquid.
    </p>
    <p
      className="p2"
    >
      Lorem ipsum dolor sit amet, consectetueradipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes , nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo. Nullam dictum felis eu pede mollis pretium. Integer tincidunt.
    </p>
  </div>
</div>
`;
